src/Api/Management/Controller/StatsController.php

```diff
@@ -68,22 +74,35 @@ public function __construct(string $environment, string $organization, ClientInt
         // Return responses as an associative array instead of in Apigee Edge's mixed object-array structure to
         // make developer's life easier.
         $this->jsonDecoder = new JsonDecode(true);
+        $this->optimized_js = false;
+        $this->is_hybrid = false;
```

Comment:
1. Properties should be camel cased
2.  Why don't you set the value here?
`ClientInterface::HYBRID_ENDPOINT === $this->getClient()->getEndpoint();` Or if that is not possible, then this should not be a property rather a helper method.

Reply:
I've moved that to separate helper method